client/views/room/contextualBar/Call/Jitsi/CallJitsi.js

```diff
@@ -0,0 +1,178 @@
+import React, { useRef, useEffect, useState, useMemo, useLayoutEffect } from 'react';
+import { Box, Skeleton } from '@rocket.chat/fuselage';
+import { useMutableCallback, useSafely } from '@rocket.chat/fuselage-hooks';
+
+import { CallModal } from './components/CallModal';
+import { useSetModal } from '../../../../../contexts/ModalContext';
+import { useUserRoom, useUser } from '../../../../../contexts/UserContext';
+import { useTranslation } from '../../../../../contexts/TranslationContext';
+import { useSettings } from '../../../../../contexts/SettingsContext';
+import VerticalBar from '../../../../../components/VerticalBar';
+import { HEARTBEAT, TIMEOUT, DEBOUNCE } from '../../../../../../app/videobridge/constants';
+import { useMethod } from '../../../../../contexts/ServerContext';
+import { useConnectionStatus } from '../../../../../contexts/ConnectionStatusContext';
+import { JitsiBridge } from './lib/JitsiBridge';
+
+export const CallJitsi = ({
+	handleClose,
+	openNewWindow,
+	refContent,
+	children,
+}) => {
+	const t = useTranslation();
+
+	const content = openNewWindow ? <>
+		<Box fontScale='p2'>{t('Video_Conference')}</Box>
+		<Box fontScale='p1' color='neutral-700'>{t('Opened_in_a_new_window')}</Box>
+	</> : <div ref={refContent}/>;
+
+	return <>
+		<VerticalBar.Header>
+			<VerticalBar.Icon name='phone'/>
+			<VerticalBar.Text>{t('Call')}</VerticalBar.Text>
+			{handleClose && <VerticalBar.Close onClick={handleClose}/>}
+		</VerticalBar.Header>
+		<VerticalBar.ScrollableContent>
+			{content}
+			{children}
+		</VerticalBar.ScrollableContent>
+	</>;
+};
+
+const querySettings = {
+	_id: [
+		'Jitsi_Open_New_Window',
+		'Jitsi_Domain',
+		'Jitsi_URL_Room_Hash',
+		'uniqueID',
+		'Jitsi_URL_Room_Prefix',
+		'Jitsi_URL_Room_Suffix',
+		'Jitsi_Chrome_Extension',
+		'Jitsi_SSL',
+		'Jitsi_Enabled_TokenAuth',
+	],
+};
+
+export default React.memo(({ tabBar, rid }) => {
```

Comment:
`React.memo((...) => ...)` will generate an anonymous component (harder to debug on DevTools) because the arrow function itself is anonymous. As an alternative, consider doing this:

```js
const ComponentWithName = (...) => ...

export default memo(ComponentWithName);
```